### PR TITLE
Integrate review feedback into Power BI baseline

### LINKED ISSUE #536: Integrate CISA review feedback into Power BI baseline
# 💡 Summary #

CISA formal review will provide feedback that may require changes to the markdown formatted Power BI baseline.

## Motivation and context ##

CISA formal review may discover changes that materially affect the baselines.  If changes are made, they should be reflected in the markdown baselines as the source of truth for secure configuration baselines.

## Implementation notes ##

Please provide details for implementation, such as:

- Review CISA feedback in terms of suggested changes and comments
- Adjudicate comments with CISA to confirm recommended changes
- Make recommended changes to the markdown baselines

## Acceptance criteria ##

How do we know when this work is done?

- [ ] CISA feedback on Power BI SCB has been received by the developers
- [ ] All CISA feedback has been reviewed and changes recommended and adjudicated successfully
- [ ] Changes have been integrated into Power BI markdown and merged into Emerald branch

### PR DESCRIPTION
**Pull request by amart241**
## 🗣 Description ##

This update includes the following changes to the Power BI security configuration baseline:

- Changes to improve overall consistency of format and language throughout
- Grammar and spelling fixes
- Language and verbiage updates for increased reader clarity
- This update does NOT include any materially significant changes to the number of or content in policy items in the baseline itself that would require a change in settings to align with the baseline recommendations.

## 💭 Motivation and context ##

The updates included further refine and improve the overall readability and clarity of the baseline document. The result should be a baseline that is easier to interpret and use as a reference for M365 tenant administrators.

Closes #536 

## 🧪 Testing ##

Compare the separately provided Word version with track changes of the Power BI baseline document with the changes in the markdown in this branch. Accepted/modified suggested changes as appropriate and mirrored in markdown, rejected and additional changes commented in Word and sync'd to markdown as needed.

Manual review of markdown and ensure that the automated markdown test on the PR passes.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Rebase feature branch against parent branch, if needed

## ✅ Post-merge checklist ##

- [ ] Confirm parent branch passes tests after merge

### CODE REVIEW

File: baselines/powerbi.md [hunk 1]
```diff
@@ -1,26 +1,27 @@
+**`TLP:CLEAR`**
+
 # CISA M365 Security Configuration Baseline for Power BI
 
-Power BI is the Software as a Service (SaaS) offering from Microsoft that facilitates self-service business intelligence dashboards, reports, datasets, and visualizations. Power BI can connect to multiple, different data sources, combine and shape data from those connections, then create reports and dashboards to share with others. This Secure Configuration Baseline (SCB) provides specific policies to strengthen Power BI security.
+Power BI is the software as a service (SaaS) product from Microsoft that facilitates self-service business intelligence dashboards, reports, datasets, and visualizations. Power BI can connect to multiple different data sources, combine and shape data from those connections, then create reports and dashboards to share with others. This Secure Configuration Baseline (SCB) provides specific policies to strengthen Power BI security.
```
Comment (mheller93766) [inline baselines/powerbi.md:5]: Change first sentence to:
Microsoft 365 (M365) Power BI is a cloud-based product that facilitates self-service business intelligence dashboards, reports, datasets, and visualizations. 

The "Microsoft 365 (M365)" is required to match the standard. The "is a cloud-based" may not apply to Power BI but is the standard phase used in the other SCBs

File: baselines/powerbi.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
+## License Compliance and Copyright
+Portions of this document are adapted from documents in Microsoft's [M365](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/LICENSE) and [Azure](https://github.com/MicrosoftDocs/azure-docs/blob/main/LICENSE) GitHub repositories. The respective documents are subject to copyright and are adapted under the terms of the Creative Commons Attribution 4.0 International license. Source documents are linked throughout this document. The United States government has adapted selections of these documents to develop innovative and scalable configuration standards to strengthen the security of widely used cloud-based software services.
```
Comment (mheller93766) [inline baselines/powerbi.md:14]: Change the third sentence to: 
Sources are linked throughout this document.

To match the standard format

**Review by mheller93766 — CHANGES_REQUESTED** [review]

File: baselines/powerbi.md
[the remark this answers is not in the document]
Comment (amart241) [reply baselines/powerbi.md:1]: Fixed.

File: baselines/powerbi.md (hunk 1 above)
Comment (amart241) [reply baselines/powerbi.md:5]: Thanks for the re-word looks good

File: baselines/powerbi.md (hunk 2 above)
Comment (amart241) [reply baselines/powerbi.md:14]: fixed

File: baselines/powerbi.md
[the remark this answers is not in the document]
Comment (amart241) [reply baselines/powerbi.md:1]: also I made the notes the `[!Note]` kind so they stick out more than the other standard note as it looks greyed out and harder to read in my opinion

**Review by mheller93766 — APPROVED** [review]

**Comment by nanda-katikaneni** [comment]
@amart241 - can you please address branch conflicts and resolve remaining comments.

File: baselines/powerbi.md [hunk 3]
```diff
@@ -614,36 +587,33 @@ For Power BI embedded applications, it is recommended to implement a
...
-such as Azure Key Vault.
+such as Azure key vault.
 
 Store encryption keys or service principal credentials used for
 encrypting or accessing Power BI in a Key Vault, assign proper access
 policies to the vault and regularly review access permissions.
 
 For regulatory or other compliance reasons, some agencies may need to
-bring their own keys (BYOK), which is supported by Power BI. By default,
+(BYOK), which is supported by Power BI. By default,
```
[the remark this answers is not in the document]
Comment (amart241) [reply baselines/powerbi.md]: Sorry I missed that. fixed now

File: baselines/powerbi.md
[the remark this answers is not in the document]
Comment (amart241) [reply baselines/powerbi.md:1]: reverted to other comment style. fixed

[the remark this answers is not in the document]
Comment (amart241) [reply baselines/powerbi.md:1]: fixed

**Review by amart241 — COMMENTED** [review]
Reviewed all changes and resolved all conversations and conflicts

**Pull request merged**